### PR TITLE
Mvvm arcthitecture

### LINKED ISSUE #1: Change to MVVM

### PR DESCRIPTION
**Pull request by vicgles**
close #1 .

### CODE REVIEW

File: app/src/main/java/com/example/desafioconcrete/Adapter/AdapterDetails.kt [hunk 1]
```diff
@@ -18,17 +18,17 @@ class AdapterDetails(
 
     override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
         var detail = detailsList[position]
-        var title = holder.itemView.lblTitleName
+        var title = holder.itemView.hint3
         var description = holder.itemView.lblDescricaoRepositorioDetails
-        var username = holder.itemView.lblUsernameDetails
+        var username = holder.itemView.hint2
```
Comment (cs-eltonsantana) [inline app/src/main/java/com/example/desafioconcrete/Adapter/AdapterDetails.kt:23]: Why are the variables called with numbers?

File: app/src/main/java/com/example/desafioconcrete/ui/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+//
+//
+//
+//                        }
+//                        super.onScrolled(recyclerView, dx, dy)
+//                    }
+//
+//
+//                })
+
```
Comment (cs-eltonsantana) [inline app/src/main/java/com/example/desafioconcrete/ui/MainActivity.kt]: If it's not running it must not go to master. Delete comments.

**Review by cs-eltonsantana — COMMENTED** [review]
Try to not leave much white lines in the code.

File: app/src/main/java/com/example/desafioconcrete/Adapter/AdapterDetails.kt (hunk 1 above)
Comment (vicgles) [reply app/src/main/java/com/example/desafioconcrete/Adapter/AdapterDetails.kt:23]: It's only for a loading activity

File: app/src/main/java/com/example/desafioconcrete/ui/MainActivity.kt (hunk 2 above)
Comment (vicgles) [reply app/src/main/java/com/example/desafioconcrete/ui/MainActivity.kt]: I'll Delete all comments

**Review by cs-eltonsantana — APPROVED** [review]
Great! Ready to go!

**Pull request merged**